waspc/src/Wasp/Generator/WebAppGenerator/Common.hs

```diff
@@ -123,3 +130,14 @@ toViteImportPath :: Path Posix (Rel r) (File f) -> Path Posix (Rel r) (File f)
 toViteImportPath = fromJust . SP.parseRelFileP . dropExtension . SP.fromRelFileP
   where
     dropExtension = fst . splitExtension
+
+getBaseDir :: AppSpec -> String
```

Comment:
I would make this one return Strong Path, (Posix) Abs Dir.

Reply:
Good idea 👍